### PR TITLE
Add tutorial documentation

### PR DESCRIPTION
**Pull request by napperley**
The **gradle-script-kotlin** repository has some documentation but lacks a decent tutorial for people to get started with **gradle-script-kotlin**. Although there are some sample projects some people prefer to learn by doing things from the ground up using their own projects.

### CODE REVIEW

File: doc/getting-started/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+#Tutorial
```
Comment (mfarsikov) [inline doc/getting-started/tutorial.md]: Please add space after `#` or `##` for correct rendering

Comment (napperley) [inline doc/getting-started/tutorial.md]: Made the adjustments. Refer to the [new commit](https://github.com/gradle/kotlin-dsl/pull/382/commits/d09fee34d7ffe6160414d2b9f9e0434e99a7b3b0).

**Comment by JLLeitschuh** [comment]
This looks really good!

File: doc/getting-started/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+tasks {
+    // Create a custom task called "createDokkaJar" that is based on the Jar type, and depends on 
+    // the "dokka" task to run first before executing.
+    "createDokkaJar"(Jar::class)" {
+        dependsOn("dokka")
+        classifier = "javadoc"
+        from(dokka.outputDirectory)
+    }
+}
+```
```
Comment (JLLeitschuh) [inline doc/getting-started/tutorial.md]: Can you add something mentioning:
```kotlin
val createDokkaJar = task<Jar>("createDokkaJar") {
    dependsOn("dokka")
    classifier = "javadoc"
    from(dokka.outputDirectory)
}
```
as also being a valid way of creating the task?

File: doc/getting-started/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+In this tutorial all the basics of using the Gradle Kotlin DSL (GKD) will be covered, which includes the following:
+
+- Specifying the basic project information
+- Including additional project meta data to use in the entire build script
+- Adding dependencies
+- Using and configuring Gradle plugins
+- Adding repositories
+
+Every Gradle project that uses GKD must include a build file called **build.gradle.kts** in the root directory of the project, and [Gradle wrapper](https://docs.gradle.org/3.5/userguide/gradle_wrapper.html) version **4.0-rc-1**.
```
Comment (JLLeitschuh) [inline doc/getting-started/tutorial.md]: Change this link to:
```
https://docs.gradle.org/current/userguide/gradle_wrapper.html
```

File: doc/getting-started/tutorial.md (hunk 2 above)
Comment (napperley) [inline doc/getting-started/tutorial.md]: Done. Refer to the new commit: https://github.com/gradle/kotlin-dsl/pull/382/commits/052dd9dd82a4fab14f8195be13365801c1f8804f

**Comment by bamboo** [comment]
First of all, thanks for PR!

Considering that this PR is outdated and the information covered here has been already covered elsewhere, in particular:
 - https://github.com/gradle/kotlin-dsl/tree/master/doc/getting-started
 - https://guides.gradle.org/building-kotlin-jvm-libraries/

Closing as outdated.

**Pull request closed without merging**